### DISCUSSION TITLE
"Downloaded" folder

### DISCUSSION THREAD

**Discussion by ghost** [civitai/civitai#489] [Ideas]
Pretty simple idea: Beside the "Like" button, there is a "Downloaded" button to keep track of what I have downloaded, so instead of getting blanket notifications between things I plan on downloading and things I already have downloaded I have an easier time keeping track of things. There are plenty of Checkpoints/Loras I want to try in the future but I would also like to keep track of which things I have that have been updated.

**Comment by octopushugger** [comment]
Until this is implemented you could just browser bookmark the urls for models you want to check out later but don't want notifications for.

**Reply by ghost** [reply]
I do a lot of phone browsing, and it's a hassle. This is just a wish list item, I'm alright at keeping everything sorted... for now :P

**Reply by JingJang** [reply]
You can look at your "Download History"? https://civitai.com/user/downloads

Am I missing something? I think you mean the actual model page has the button that says "Downloaded Already". I think that would make more sense.

**Reply by ghost** [reply]
You don’t know me, I am very dumb. I did not see that button because I don’t read. 🤣
